scripts/automator/build/compiler-defaults

```diff
@@ -1,10 +1,11 @@
 # Tools and flags for all compilers
-VARIABLES=(ar cc cxx ld ranlib cflags cxxonly ldflags libs)
+VARIABLES=(ar cc cxx ld ranlib arflags cflags cxxonly ldflags libs)
 ar=""
 cc=""
 cxx=""
 ld=""
 ranlib=""
+arflags="cr"
```

Comment:
Please change the flag combination to `Dcr`. It will result in slightly slower builds, but I prefer for us to be closer to reproducible builds on any OS than save a fraction of a second per build (as our `.a` files are rather small).

Reply:
Updated; thanks @dreamer !
`D` is the default, but might as well be explicit if it's not default in some implementations.